### PR TITLE
Closes #296 - Implement functionality that is required for sending information from Contact us form on web to an e-mail adress that should be specified in a configurational file

### LINKED ISSUE #296: Implement functionality that is required for sending information from Contact us form on web to an e-mail adress that should be specified in a configurational file

### PR DESCRIPTION
**Pull request by maxturko**

### CODE REVIEW

**Comment by maxturko** [comment]
Need to rewrite LOGGER input values and method descriptions in the interface

File: src/main/java/com/softserve/academy/spaced/repetition/controller/dto/simpleDTO/MailDTO.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
+package com.softserve.academy.spaced.repetition.controller.dto.simpleDTO;
```
[the remark this answers is not in the document]
Comment (antonbakalets) [reply src/main/java/com/softserve/academy/spaced/repetition/controller/dto/simpleDTO/MailDTO.java]: The best solution would be to create a separate PR that just do the renaming.

File: src/main/java/com/softserve/academy/spaced/repetition/service/MailService.java [hunk 2]
```diff
@@ -69,4 +76,12 @@
      */
     String getRestorePasswordTemplateContent(Map<String, Object> model);
 
+    /**
+     * Generates the text of email with details of user's request which we get via contact us form.
+     *
+     * @param model map with values which will be inserted into html page.
+     * @return content of the email.
+     */
+    String contactUsMailTemplateContent(Map<String, Object> model);
```
Comment (antonbakalets) [inline src/main/java/com/softserve/academy/spaced/repetition/service/MailService.java]: As I understand this method is a private method of service implementation (MailServiceImpl.java). There is no need to expose it through the interface.
If something can be private - make it private (encapsulation).

**Review by antonbakalets — CHANGES_REQUESTED** [review]

File: src/main/java/com/softserve/academy/spaced/repetition/controller/dto/simpleDTO/MailDTO.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (maxturko) [reply src/main/java/com/softserve/academy/spaced/repetition/controller/dto/simpleDTO/MailDTO.java]: Do I need to propose changes to product owner in this case ?

[the remark this answers is not in the document]
Comment (antonbakalets) [reply src/main/java/com/softserve/academy/spaced/repetition/controller/dto/simpleDTO/MailDTO.java]: No, as such it doesn't change the application behaviour.

**Review by antonbakalets — APPROVED** [review]

**Review by OleksandrBartosh — APPROVED** [review]

**Review by volodymyrglynka — APPROVED** [review]

**Review by hlibOstapenko — APPROVED** [review]

**Pull request merged**